CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+ * `TimelockController`: During deployment, the TimelockController used to grant the `TIMELOCK_ADMIN_ROLE` to the `admin` parameter, it will now assign the `DEFAULT_ADMIN_ROLE` to the `admin` parameter to be consistent with the rest of the repository.([#3799](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3799))
```

Comment:
This needs to be under yet another (:sweat_smile:) "Unreleased" section at the top... Let's name it "Unreleased (Breaking)" so it's less ambiguous for now.

Also, there's a missing space after the period.